Orange/widgets/unsupervised/owpca.py

```diff
@@ -196,9 +200,16 @@ def set_data(self, data):
             self.sampling_box.setVisible(False)
         if isinstance(data, Table):
             if data.is_sparse():
-                self.Error.sparse_data()
-                self.clear_outputs()
-                return
+                # PCA does not support sparse data
+                # Falling back to TruncatedSVD aka LSA
+                self.center = False
```

Comment:
Besides `self.center` also `self.normalize` shoud be disabled and set to False for sparse data sets.

Default normalization cannot handle sparse data set. It might have worked on BoW data (for which all features are marked to be skipped during normalization), but it does not work for sparse data sets in general.

To reproduce the problem, transform Iris to sparse in Python script, pass it to PCA and check Normalize data.